### PR TITLE
Redesign of the newsletter section!

### PR DESCRIPTION
**Pull request by tobyab**

### CODE REVIEW

**Review by grymmy — CHANGES_REQUESTED** [review]
It seems to me that there are a large number of changes in this PR relating to formatting that do not relate to the actual redesign that was the cause for the PR in the first place.  Can these formatting changes be done as a separate diff to clearly delineate where the feature begins/ends vs the other changes to ;'s, "'s, and whitespace?

**Comment by tobyab** [comment]
Hey @grymmy! Just removed the formatting.

**Review by grymmy — APPROVED** [review]
Lgtm @maxwofford I believe you were gonna do a design pass

**Comment by tobyab** [comment]
@maxwofford have also fixed corner radius + padding issues!

File: components/index/cards/mailing-list.js [hunk 1]
```diff
@@ -1,60 +1,53 @@
...
-  Text,
-  Alert,
-  Card,
-  Flex,
-  Grid
-} from 'theme-ui'
-import CardModel from './card-model'
-import BGImg from '../../background-image'
-import FooterImgFile from '../../../public/home/footer.png'
+import Icon from "@hackclub/icons";
```
Comment (maxwofford) [inline components/index/cards/mailing-list.js]: Did you mean to switch this to single quote, semicolon lines?

File: components/mail-card.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+          <Box
+            sx={{
+              placeItems: "center",
+              display: "grid",
+              height: "100%",
+              paddingY: [3, 4, 0],
+            }}
+          >
+            <Box sx={{ px: [3, 4] }}>
+              <Box>
```
Comment (maxwofford) [inline components/mail-card.js]: Lots of nesting in this area-- are all these `<Box>`s needed?

File: components/index/cards/mailing-list.js [hunk 3]
```diff
@@ -1,60 +1,53 @@
...
+import { useEffect, useRef, useState } from "react";
+import { Box, Button, Card, Flex, Grid, Input, Link, Text } from "theme-ui";
+import BGImg from "../../background-image";
+import background from "../../../public/home/footer.png";
+import MailCard from "../../mail-card";
+import { format } from "date-fns";
```
Comment (maxwofford) [inline components/index/cards/mailing-list.js]: Most of our pages list local files imports below npm imports

**Review by maxwofford — APPROVED** [review]
Added a couple small styling comments, but overall this is approved for merge!

**Comment by maxwofford** [comment]
Awesome! Go for merge

**Pull request merged**